### DISCUSSION TITLE
Regulating/Improving the Twitter Spaces.

### DISCUSSION THREAD

**Discussion by manavmodi22** [EddieHubCommunity/support#1167] [Ideas]
Twitter Spaces have enormous potential and planning a more organized initiative could help putting it to the right use!

**Comment by eddiejaoude** [comment]
Good idea 👍 some of my suggestions...

- schedule / calendar of events
- record TwitterSpaces for podcast
- limit each person to speak to 1min with a timer, will give more people time to speak
